packages/datetime2/test/components/dateRangePicker3Tests.tsx

```diff
@@ -237,13 +237,13 @@ describe("<DateRangePicker3>", () => {
 
             it("calls onDayMouseEnter", () => {
                 const onDayMouseEnter = sinon.spy();
-                render({ defaultValue, dayPickerProps: { onDayMouseEnter } }).left.mouseEnterDay(14);
```

Comment:
Why weren't these caught before? I assume that we weren't running the tslint checks on these tests?

Reply:
The previous lint rule did not check objects that were contained to a single line, it only ran when the object spanned multiple lines. Most of the changes in this PR result from that difference in rule behavior.